### DISCUSSION TITLE
Inconstistence between Verifier.VerifyXml and Verifier.VerifyJson

### DISCUSSION THREAD

**Discussion by lechu445** [VerifyTests/Verify#955] [General]
Verifier.VerifyXml produces `.xml` files and Verifier.VerifyJson produces not `.json` files but `.txt`. I'd say that's pretty surprising API design decision; I'd expect that either both are treated as a text, or both are saved in its own format.

This comes to bigger issue, **Verifier.VerifyJson does not even allow to save file as json**, or set custom extension of the file.

Example:
```c#
[UsesVerify]
public class UnitTest
{
    [Fact]
    public async Task VerifyXml() => await Verifier.VerifyXml("<a/>");

    [Fact]
    public async Task VerifyJson() => await Verifier.VerifyJson("{}");
}
```
produce:
- UnitTest.VerifyJson.received.txt
- UnitTest.VerifyJson.verified.txt
- UnitTest.VerifyXml.received.xml
- UnitTest.VerifyXml.verified.txt

**Comment by SimonCropp** [comment]
https://github.com/VerifyTests/Verify/blob/main/docs/serializer-settings.md#usestrictjson
